inspector/src/pages/utils.ts

```diff
@@ -98,3 +98,72 @@ export function createClearStoredConfigButton(
   }
   return buttonElt;
 }
+
+export function createTimeRepresentationSwitch(
```

Comment:
Can you add the TypeScript return type so it's clearer when reading this what this does?
Same for other utils